src/libcore/ops.rs

```diff
@@ -166,25 +166,38 @@ macro_rules! forward_ref_binop {
 ///
 /// # Examples
 ///
-/// A trivial implementation of `Add`. When `Foo + Foo` happens, it ends up
-/// calling `add`, and therefore, `main` prints `Adding!`.
+/// This example creates a Point struct that implements Add, and then
+/// demonstrates adding two Points.
```

Comment:
I think it's better this way: "demonstrates adding two `Point`s". Or "two `Point` structs".